### DISCUSSION TITLE
[REQUEST] Add OpenYoutube Extension and Filesbackup

### DISCUSSION THREAD

**Discussion by bossx2** [qnblackcat/uYouPlus#268] [General]
OpenYoutube extension makes safari open videos on sideloaded youtube and filesbackup helps you backup to the files app

**Comment by qnblackcat** [comment]
Open in YouTube extensions has been included for a while
I‘m not sure what is “file backup extension”? What repo is it?

**Reply by bossx2** [reply]
I honestly don’t know the repo at this moment I will search for it but it basically backups all the apps files to the ios files app in case the app gets revoked then you can easily import them again to the app I have seen it used a lot in twitter and thought it’s a seriously cool feature to add I will reply back when I get the repo
